lbuild/environment.py

```diff
@@ -203,6 +204,8 @@ def template(self, src, dest=None, substitutions=None, filters=None):
             else:
                 dest = src
 
+        src = self.repopath(src)
```

Comment:
Yes, good point, I'll add it to copy then too:
```
ERROR: Cannot access template outside of repository!
../board.cpp.in
```
Btw, is it possible to print the module name from which the exception was thrown? Otherwise good luck finding the problem.